### ISSUE TITLE
Alternate list background color for Notes section

### ISSUE THREAD

**Issue by rimma068** [Kunzisoft/KeePassDX#1163]
I would like to have an option to alternate list background color for Notes. It is hard to distinguish where one line ends and a new line starts when the screen size is small. For example, when lines start and end with numbers, it is hard to tell which numbers belong to which lin
![123](https://user-images.githubusercontent.com/13087157/142720871-0e865133-a0f2-4088-9f80-f3d90e50254b.PNG)
e.

**Comment by J-Jamet** [comment]
Good idea, I don't know if it's easily done on an EditText but it's worth considering.
